app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInActivity.cs

```diff
@@ -113,6 +114,9 @@ private void InitButtonsAndInputs()
             registerButton.Click += (sender, e) =>
             {
                 //StartActivityForResult(new Android.Content.Intent(this, typeof(RegisterActivity)), REQUEST_CODE_REGISTER);
```

Comment:
Either use the suggestion I made regarding activity flow in RegisterActivity and uncomment this code, or delete it